contracts/governance/Vesting/fouryear/FourYearVestingLogic.sol

```diff
@@ -137,6 +138,42 @@ contract FourYearVestingLogic is IFourYearVesting, FourYearVestingStorage, Appro
 		emit DividendsCollected(msg.sender, _loanPoolToken, _receiver, _maxCheckpoints);
 	}
 
+	/**
+	 * @notice Change token owner - only vesting owner.
+	 * @dev Modifies token owner. This must be followed by approval
+	 * from token owner.
+	 * @param _newTokenOwner Address of new token owner.
```

Comment:
should it be multisig only? if yes then pls rename the variable to _newTokenOwnerMultisig and at least add verification that the new address is a contract.

Reply:
New token owner can be any other user